### DISCUSSION TITLE
报告一个校对 bug

### DISCUSSION THREAD

**Discussion by PirateBook1** [banned-historical-archives/banned-historical-archives.github.io#2493] [Q&A] [answered]
@banned-historical-archives 

当文稿未校对时（如下图），存在一些为空的段落：
![image](https://user-images.githubusercontent.com/113572850/221750242-5c0f963a-cba4-4c3e-af24-d5d8d7f0354c.png)

此时在内部加上一些文字：
![image](https://user-images.githubusercontent.com/113572850/221750363-d2ad46b3-c686-4f82-9f70-521edadb6ac6.png)

这个更改并没有在 issue 中反映出来：
![image](https://user-images.githubusercontent.com/113572850/221750513-04d1a138-838e-4460-b7e6-977b965d376a.png)

**Comment by PirateBook1** [comment, marked as answer]
@banned-historical-archives 这个 bug 非常严重，可能意味着以往的所有校对或多或少都缺失了段落

**Reply by PirateBook1** [reply]
或者是我校对方式有误，这样的空白段落不应该填写东西而应该勾选删除？

**Reply by banned-historical-archives** [reply]
我应该在写入数据库前过滤掉空白段落

**Reply by banned-historical-archives** [reply]
我可以修复这个问题，可能会影响42篇已校对的文稿，到时麻烦再检查一下。
以下是原始解析后的文稿存在空白段落 且校对过的文稿

$$$ 重要讲话 在关于知识分子问题会议上的讲话
$$$ 重要讲话 事情正在起变化
$$$ 重要讲话 陈伯达等对“赴广州揪王任重革命造反团”的指示
$$$ 共产党宣言（人民出版社1963年版） 共产党宣言
$$$ 上海市机电一局系统活学活用毛泽东思想积极分子代表大会上市革会领导成员、工总司负责人王洪文同志的讲话 上海市机电一局系统活学活用毛泽东思想积极分子代表大会上市革会领导成员、工总司负责人王洪文同志的讲话
$$$ 毛远新一九七四年一月在团省委召开的“学习吴献忠座谈会”上的讲话（节录） 毛远新一九七四年一月在团省委召开的“学习吴献忠座谈会”上的讲话（节录）
$$$ 关于日本问题的报告（1956.11） 张春桥在上海市委宣传部第十三次时事政策讲演会上的报告
$$$ 姚文元同志在欢庆黄浦区革命委员会成立大会上的讲话 姚文元同志在欢庆黄浦区革命委员会成立大会上的讲话
$$$ 中央首长第三次接见安徽代表（1967.9.5） 中央首长第三次接见安徽双方代表团的指示
$$$ 张春桥同志二月二十五日与华东局革命造反派座谈纪要 张春桥同志与华东局革命造反派座谈纪要
$$$ “四人帮”罪行材料（三）（1976.12） 王洪文在总参民兵组训工作座谈会议上的讲话
$$$ 王洪文副主席在接见总参民兵组训工作座谈会议全体同志时的讲话 王洪文副主席在接见总参民兵组训工作座谈会议全体同志时的讲话
$$$ 毛远新在省工农兵干部学习班的几次讲话（摘编） 毛远新在省工农兵干部学习班的讲话（节录）
$$$ 在工总司召开的“坚决响应毛主席伟大号召，‘斗私批修’誓师大会”上张春桥同志的讲话 在工总司召开的“坚决响应毛主席伟大号召，‘斗私批修’誓师大会”上张春桥同志的讲话
$$$ 揭发“四人帮”王洪文插手控制上影阴谋篡党夺权的罪行 第二部分
$$$ 张春桥同志讲话 张春桥同志讲话
$$$ “四人帮”死党毛远新破坏教育革命言论摘编（初编） 四、破坏党的一元化领导，削弱工宣队的政治作用
$$$ 简明世界史（近代部分）（北京大学历史系1974年版） 第一章 英国资产阶级革命 美国独立战争
$$$ 简明世界史（近代部分）（北京大学历史系1974年版） 第四章 十七——十九世纪初亚非人民反对殖民主义、封建主义的斗争
$$$ 简明世界史（近代部分）（北京大学历史系1974年版） 插图
$$$ 红旗一九五八年第一期 一个苦战了两年改变了面貌的合作社
$$$ 红旗一九五八年第一期 南斯拉夫修正主义是帝国主义政策的产物
$$$ 红旗一九五八年第一期 劳动人民一定要做文化的主人
$$$ “四人帮”罪行材料（四）（1976.12） 王洪文、张春桥在作战部汇报批林批孔运动情况时的插话
$$$ 毛远新有关研究“儒法斗争史”的三次谈话（摘编） 一、毛远新一九七四年七月七日在省委批林批孔汇报会议上的总结（节录）
$$$ 毛远新有关研究“儒法斗争史”的三次谈话（摘编） 三、毛远新一九七四年十月十二日在沈阳机床一厂汇报运动情况时的讲话（节录）
$$$ 毛远新在批邓、反击右倾翻案风斗争中的一些言论（摘编） （一）毛远新在反击右倾翻案风开始时的讲话（摘录）
$$$ 毛远新在批邓、反击右倾翻案风斗争中的一些言论（摘编） 毛远新的谈话记录
$$$ 夏邦银、朱洪霞、胡厚民等人的言论节录 给伟大领袖毛主席的一封信
$$$ 夏邦银、朱洪霞、胡厚民等人的言论节录 朱洪霞等人在市建工局礼堂集会上的讲话（节录）
$$$ 夏邦银、朱洪霞、胡厚民等人的言论节录 朱洪霞、胡厚民在江汉工人文化宫集会上的讲话（节录）
$$$ 夏邦银、朱洪霞、胡厚民等人的言论节录 顾建棠等人在市建工局礼堂集会上的讲话（节录）
$$$ 夏邦银、朱洪霞、胡厚民等人的言论节录 朱洪霞在国棉六厂子弟小学集会上的讲话（节录）
$$$ 张春桥同志十一月二十五日在市革会扩大会议上的讲话纪要（1968.11.25） 张春桥同志在上海市革命委员会扩大会议上的讲话
$$$ 张春桥关锋同志接见福建部分赴京革命同学的讲话 张春桥关锋同志接见福建部分赴京革命同学的讲话
$$$ “四人帮”罪行材料（一〇）（1977.1） 王洪文、张春桥在中央军委领导同志听取通信兵批林批孔运动情况汇报时的插话
$$$ 姚文元关于国际形势的报告（1967.6） 姚文元同志关于国际形势的报告
$$$ 洪文、春桥同志批示 上海市委领导同志在警备区看大字报时讲话
$$$ 洪文、春桥同志批示 关于许世友的大字报
$$$ 毛远新的谈话记录 毛远新的谈话记录
$$$ 毛远新破坏民兵建设的部分罪行材料 （八）毛远新一九七四年十一月二十日在辽、吉、黑三省民兵工作座谈会上的讲话（摘录）
$$$ 王洪文、纪登奎同志在省委工作会议召集人和县委书记会议上的讲话（1975.7.9） 王洪文、纪登奎在省委工作会议召集人和县委书记会议上的讲话

**Reply by PirateBook1** [reply]
改吧

**Reply by banned-historical-archives** [reply]
现在应该修好了。但还是要检查一下，我们分批吧，你前面21篇，我后面21篇。

**Reply by PirateBook1** [reply]
@banned-historical-archives 不用分批，我来就好，每个已校对 page 我都有本地保存
